server.js

```diff
@@ -1,33 +1,68 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
+import express from "express";
+import bodyParser from "body-parser";
+import cors from "cors";
+import data from "./data/netflix-titles.json";
+
+// Defines the port the app will run on. Defaults to 8080, but can be
 // overridden when starting the server. For example:
 //
 //   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+const port = process.env.PORT || 8080;
+const app = express();
+const listEndpoints = require("express-list-endpoints");
+
+// Root endpoint
+app.get("/", (req, res) => {
+  res.send(listEndpoints(app));
+});
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
+
+//Displays all awards
+app.get("/", (req, res) => {
+  res.json(data);
+});
+
+// Displays all the movies/shows
+app.get("/nominations", (req, res) => {
+  res.json(data);
+});
+
+//Displays movies/shows depending on year
+app.get("/year/:year", (req, res) => {
+  const year = req.params.year;
+  const showType = req.query.type;
+  let showsYear = data.filter((item) => item.release_year === +year);
+
+  if (showType) {
+    showsYear = showsYear.filter(
+      (item) => item.type.toLowerCase() === showType.toLowerCase()
+    );
+  }
+  res.json(showsYear);
+});
+
+//Displays movies/shows from a specific country
+app.get("/countries/:countries", (req, res) => {
+  const countries = req.params.countries;
+
+  let showCountry = data.filter(
+    (item) => item.country.toString().toLowerCase() === countries.toLowerCase()
+  );
+
+  res.json(showCountry);
+});
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+//Displays a movie/show with a specific show_id
+app.get("/showId/:showId", (req, res) => {
```

Comment:
Because all your shows are listed in `/nominations`, so to be 100% REST this path to one show should be `/nominations/:showId`.

So if you like to change nominations to shows to be less golden globe and more Netflix ;) you should make this path `/shows/showId`